doc/ref/chap-builtin-func.md

```diff
@@ -32,8 +45,66 @@ This chapter in the [Oils Reference](index.html) describes builtin functions.
 
 ### sum()
 
-## Int
+## Bool()
+
+Returns the truth value of its argument. Similar to `bool()` in python, it returns `false` for `false`, `0`, `0.0`,
+`''`, `{}`, `[]`, and `null`.  Returns `true` for all other values.
+
+## Int()
+
+Given a float, returns the largest integer that is less than the given value.
+
+    $ = Int(1.23)
```

Comment:
Just to be extra explicit,   maybe change this to 1.99 and mention floor(), since I think we also want the round() function, which has the "other" behavior